subsys/logging/log_output.c

```diff
@@ -166,8 +167,8 @@ static void color_postfix(struct log_msg *msg,
 }
 
 
-static int ids_print(struct log_msg *msg,
-		     const struct log_output *log_output, bool level_on)
+static int ids_print(struct log_msg *msg, const struct log_output *log_output,
+		     bool level_on, bool func_on)
```

Comment:
One of next releases, you'll probably want to replace this with a single "flags" param.